kernel_gateway/tests/test_gatewayapp.py

```diff
@@ -39,8 +39,6 @@ def test_config_env_vars(self):
         os.environ['KG_SEED_URI'] = 'fake-notebook.ipynb'
         os.environ['KG_PRESPAWN_COUNT'] = '1'
         os.environ['KG_DEFAULT_KERNEL_NAME'] = 'fake_kernel'
-        os.environ['KG_LIST_KERNELS'] = 'True'
-        os.environ['KG_ALLOW_NOTEBOOK_DOWNLOAD'] = 'True'
```

Comment:
Discussed briefly offline. These would have to be in the mode specific test cases now and not clear if the same checks can be performed there. 

I'm OK leaving them off. We know in general that the env var -> traitlet feature works which was the original purpose of the test case, no necessarily to check it for every option.